--- mcmc.py	2010-12-03 14:15:37.000000000 +1000
+++ /export/home/denhamrj/lib/python/pymcmc/mcmc.py	2010-12-06 10:25:03.432299000 +1000
@@ -1564,15 +1564,15 @@
                     for i in range(2):
                        print  >>destination, \
                         "{0: >0{colwidth}}".format("NA", colwidth = colwidth - 1),
-                    summary_vals[2] = nan
-                    summary_vals[3] = nan
+                    summary_vals[2] = np.nan
+                    summary_vals[3] = np.nan
                 if ifactor[position[0]].dtype == np.dtype('|S2'):
                     ## then not numeric.
                     ## Is there a better way of testing?
                     print  >>destination,     \
                           "{val1: >0{colwidth}}".format(val1 = ifactor[position],
                                                         colwidth = colwidth)
-                    summary_vals[4] = nan
+                    summary_vals[4] = np.nan
                 else:
                     print >>destination,      \
                           "{val1: >0{colwidth}.{sigfigs}g}".format(
@@ -1735,5 +1735,6 @@
                 print >>destination, "Log likelihood = ", LOGLIKE
                 self.output_dictionary['BIC'] = BIC
                 self.output_dictionary['LOGLIKE'] = LOGLIKE
+
                 
                 
